src/spec-transform-auto-layout.js

```diff
@@ -139,12 +146,20 @@ var getMaxTickLabelSize = function (domainValues, formatter, fnCalcTickLabelSize
         return size;
     }
 
-    var maxXTickText = _.max(domainValues, (x) => formatter(x).toString().length);
+    var maxXTickText;
+    let lastComputed = -Infinity;
```

Comment:
Sorting is lest performant. My loop is O(n) while any sorting algorithm might get to O(n^2) in the worst case.

Other option would be something like:
```    
     var maxXTickText = domainValues.reduce((prev, value) => {
         let computed = formatter(value).toString().length;
         if (!value) {
             return { value, computed };
         }
         if (computed > prev.computed) {
             return { value, computed };
         }
     }, undefined).value;
```

Would that be better?
If you still prefer the sorting option. I'll just change it :)